### DISCUSSION TITLE
Changing/Resizing the widgets

### DISCUSSION THREAD

**Discussion by XynoxTheDev** [anuraghazra/github-readme-stats#1075] [Q&A] [answered]
Can we make the size of both the stats and language stats widget size equal or resize them?

![image](https://user-images.githubusercontent.com/51535331/119156410-d60a4480-ba71-11eb-94a0-08e6b9ba14d4.png)

**Comment by florianbussmann** [comment, marked as answer]
You can resize the language stats widget to match the personal stats widget as demonstrated here: https://github.com/anuraghazra/github-readme-stats/issues/1060#issuecomment-841787919

The personal stats cannot be resized with the current codebase afaik

**Reply by XynoxTheDev** [reply]
I'm not asking about changing the components like hiding them, I want is there way to resize them. Also, can we use HTML to resize?

**Reply by florianbussmann** [reply]
The thread I linked is not about hiding components, but about changing (resizing) the card width so that it matches the other card.

**Reply by XynoxTheDev** [reply]
Ty for helping and responding.

**Reply by rickstaa** [reply]
You can also check out #1170, which request the ability to change the width of the stats card. PR #1334 will add this feature.
